### ISSUE TITLE
Tab characters are not correctly sent to terminal on windows

### ISSUE THREAD

**Issue by gitboy16** [haberdashPI/terminal-polyglot#7]
TLDR from discussion below: this appears to be an issue specific to windows, but it is not clear why it is happening yet. Package maintainer does not have access to windows. Requires someone who has windows to debug the issue.

Hi,
I noticed that tab spaces are not sent to the terminal when executing a command.
Example: (i am writing from a phone so I do not have the <tab> button on the keyboard)
`x = "hello<tab key>world"` is not properly sent to the terminal...the tab space is removed.
This happens with R, Python or Julia.
Kind regards

**Comment by haberdashPI** [comment]
Interesting... thanks for letting me know. I will see if I can track down the problem.

**Comment by haberdashPI** [comment]
I cannot reproduce this bug. I seem to be able to send tabs to my terminal just fine.

Can you provide detailed steps to replicate your issue?

**Comment by haberdashPI** [comment]
One possibility; if you are not using bracketed paste mode (see `README.md`) it could be that these three terminals are interpreting the tab key as an attempt to autocomplete. Seems unlikely it would work that way for me and not you, and the same in all three REPLs for you, but just trying to make sense of why we would be seeing different behavior.

**Comment by gitboy16** [comment]
I think the issue is that when the above is created in VS Code, it works although what is passed to the terminal is not "tab" or "\t" but spaces characters. Now, if you create the same in Notepad++ , then copy it to a .jl file then try to execute to the terminal, you should see that the tab character is ignored and you get "helloworld" in the Julia REPL.

**Comment by haberdashPI** [comment]
I made sure that the text I sent included an actual tab character (I have VSCode display whitespace with a marker to show whether whitespace is a tab or a space).

**Comment by haberdashPI** [comment]
Do you have bracketed paste mode on or not? Can you show me your settings for terminal polyglot (or are you using the default settings?)

**Comment by gitboy16** [comment]
Auto complete on or off does not change anything. I will send you my settings tomorrow. Thank you.

**Comment by gitboy16** [comment]
Here are my settings:
```
"workbench.colorTheme": "Monokai",
"security.workspace.trust.untrustedFiles": open",
"[julia]":{
  "terminal-polyglot.launchCommand": "julia",
  "terminal-polyglot.bracketPasteMode": true
},
"window.zoomlevel": -1
```

**Comment by haberdashPI** [comment]
Mm... okay. 😕 Still can't reproduce. Even with exactly these settings I have no problem sending tabs. What version of VSCode are you using? What OS? What terminal?

**Comment by gitboy16** [comment]
I am using the last version of vscode on windows with cmd.exe.

**Comment by haberdashPI** [comment]
My best guess is that this is an issue specific to cmd.exe, which I do not have available to me for debugging (since I don't have windows). 

You might want to give PowerShell a shot, I'm told it is a much better experience for terminal-based workflows than cmd.exe. Sorry, none of this is something I have much experience with so I can't provide much guidance.

**Comment by gitboy16** [comment]
I will try with powershell and let you know. Thank you.

**Comment by gitboy16** [comment]
Same issue with PowerShell.

**Comment by haberdashPI** [comment]
Okay, thanks for the information. If I ever get my hands on a windows machine I can try some of these things out for myself and try to debug the issue.

**Comment by haberdashPI** [comment]
Oh, one other question: have you tried sending tabs with bracketed paste mode *off*. I now remember that windows sometimes handles that setting poorly.

**Comment by gitboy16** [comment]
Yes I tried it does not work. I will check this weekend if I have the same issue on my mac. Thank you.
